configuration/outbound.md

```diff
@@ -0,0 +1,201 @@
+# Outbound push [beta]
```

Comment:
Mark for outbound is just the name of the transition (because that's the action the transition is performing): the schedule is called `outbound`, as is the config parameter.